cognite_toolkit/cdf_tk/load.py

```diff
@@ -521,6 +521,10 @@ def load_datamodel(
             cognite_resources_by_type[type_].append(
                 resource_cls.load(load_yaml_inject_variables(file, ToolGlobals.environment_variables()))
             )
+    # Remove duplicates
```

Comment:
Should this result in a warning? How come we end up with duplicates? The same module mentioned twice in local.yaml?